### ISSUE TITLE
Lost 1 hour of work after gradient scales up to a massive size, (Not enough RAM).

### ISSUE THREAD

**Issue by SkizzorsPS** [photopea/photopea#3422]
I spent 1 hour making a solar system just to see what it was like, as I was making the final touches with gradients an crap the gradient I was currently using scaled up to a massive size. I tried to scale it back down but was met with the error: Not enough RAM (Need 2221 MB). I want this bug fixed in the future so that more of my time is not wasted.

**Comment by MichalSZZ** [comment]
I have one hint for you that I use when I am doing something important for me:
After creating a new project, save your work, close file and open it again. Now you can use Ctrl+S in Chrome browser. Use it very oft. 
Even in Photoshop I have crashes. Photopea can't lnow how much RAM do you have and what you will do with your project (how many pixels it will have and how many layers and effects). Browsers has it own limits for RAM and the browser kill Photopea, because of RAM limit.

**Comment by photopea** [comment]
What do you mean by "gradient scaled up"? Do you mean using the Free Transform? I would like to reproduce it.

If you open Photopea and start duplicating a layer, it will always crash at some point (it could be at 1000, 10000 or 100000 copies of a layer, as your computer would run out of memory).

**Comment by SkizzorsPS** [comment]
> What do you mean by "gradient scaled up"? Do you mean using the Free Transform? I would like to reproduce it.

I was editing an object with an outer glow applied in transform mode (whatever the top one is called) while I was scaling it up I might have done something like accidentally hit a keybimd or something because I blinked and it was huge. I tried scaling it down and it said the error. I then tried deleting it and it showed the error again. I remade the artwork as well
